test/cachingTest.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+var chai = require( 'chai' );
```

Comment:
our linting allows spaces or no spaces inside parens and I don't mind either (we should probably look to consolidate to 1 method) but can we be consistent in each file please